src/plugins/discover/public/components/discover_grid/discover_grid.scss

```diff
@@ -40,6 +40,30 @@
 }
 
 .dscDiscoverGrid__cellPopover {
+  min-height: 140px;
+  max-width: 900px !important;
+
+  &.dscDiscoverGrid__cellPopover--withJson {
+    resize: horizontal;
+    min-width: 516px;
+    width: 516px;
+  }
```

Comment:
I definitely think, even if we decide not to allow resizing, having the JSON view in a larger dimension makes much sense